### PR TITLE
f1160 nextjs13

### PR DESCRIPTION
**Pull request by ssickles**
Updated this quickstart to use nextjs 13 (which was recently released at their conference).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,6 +31,6 @@ yarn-error.log*
 # vercel
 .vercel
 
-# IDEs
```
Comment (BSick7) [inline .gitignore]: Why remove these?

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nextjs-quickstart",
+  "name": "nextjs-quickstart2",
```
Comment (BSick7) [inline package.json]:
```suggestion
  "name": "nextjs-quickstart",
```

File: package.json [hunk 3]
```diff
@@ -9,12 +9,10 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "next": "12.2.2",
+    "eslint": "8.27.0",
```
Comment (BSick7) [inline package.json]: Should keep these as `devDependencies`.

File: next.config.js [hunk 4]
```diff
@@ -1,7 +1,6 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-  swcMinify: true,
```
Comment (BSick7) [inline next.config.js:4]: Did this change in nextjs 13?

File: .gitignore (hunk 1 above)
Comment (ssickles) [reply .gitignore]: accidental

File: next.config.js (hunk 4 above)
Comment (ssickles) [reply next.config.js:4]: yes, this is on by default

**Pull request merged**